src/types/simpleEarn.ts

```diff
@@ -0,0 +1,118 @@
+export interface SimpleEarnProductListParams {
+  asset?: string;
+  current?: number;
+  size?: number;
+}
+
+export interface SimpleEarnFlexibleProduct {
+  asset: string;
+  latestAnnualInterestRate: string;
+  tierAnnualPercentageRate: Record<string, number>;
+  airDropPercentageRate: string;
+  canPurchase: boolean;
+  canRedeem: boolean;
+  isSoldOut: boolean;
+  hot: boolean;
+  minPurchaseAmount: string;
+  productId: string;
+  subscriptionStartTime: number;
+  status: string;
+}
+
+export interface SimpleEarnFlexibleProductListResponse {
+  rows: SimpleEarnFlexibleProduct[];
+  total: number;
+}
+
+export interface SimpleEarnLockedProduct {
+  projectId: string;
+  detail: {
+    asset: string;
+    rewardAsset: string;
+    duration: number;
+    renewable: boolean;
+    isSoldOut: boolean;
+    apr: string;
+    status: string;
+    subscriptionStartTime: number;
+    extraRewardAsset: string;
+    extraRewardAPR: string;
+  };
+  quota: {
+    totalPersonalQuota: string;
+    minimum: string;
+  };
+}
+
+export interface SimpleEarnLockedProductListResponse {
+  rows: SimpleEarnLockedProduct[];
+  total: number;
+}
+
+export interface SimpleEarnSubscribeProductParams {
+  productId: string;
+  amount: number;
+  autoSubscribe?: boolean;
+  sourceAccount?: 'SPOT' | 'FUND' | 'ALL';
+}
+
+export interface SimpleEarnSubscribeFlexibleProductResponse {
+  purchaseId: string;
+  success: boolean;
+}
+
+export interface SimpleEarnSubscribeLockedProductResponse {
+  purchaseId: string;
+  positionId: string;
+  success: boolean;
+}
+
+export interface SimpleEarnRedeemParams {
+  positionId: string;
+}
+
+export interface SimpleEarnRedeemResponse {
+  success: boolean;
+  redeemId: string;
+}
+
+export interface SimpleEarnFlexibleProductPositionParams {
+  asset?: string;
+  productId?: string;
+  current?: number;
+  size?: number;
+}
+
+export interface SimpleEarnLockedProductPositionParams
+  extends SimpleEarnFlexibleProductPositionParams {
+  positionId?: string;
+}
```

Comment:
Since it's only 4 extra params (not 10-50), I would prefer a flatter view, even if it means a little more repetition:
```suggestion
export interface SimpleEarnLockedProductPositionParams {
  asset?: string;
  productId?: string;
  current?: number;
  size?: number;
  positionId?: string;
}
```

What might start with one simple `extends` can quickly become a mess that's hard to follow around (for some).